language-design.md

```diff
@@ -70,7 +291,49 @@ what ways do the themes apply to the study and creation of DSLs?
 
 **Response**
 
-
+There are clearly stereotypes about certain DSLs. For example, Yang and Rabkin
+mention the tendency of back-end developers to look down on front-end
+programmers. They mention Javascript in this context. While Javascript is
+clearly a general purpose language, there are DSLs involved in web development
+that also contribute to the stereotypes. For example, HTML fits our definition
+of a DSL, and more generally a programming language. It has syntax and
```

Comment:
Calling HTML a programming language is a huge stretch.  The vast number of articles you mention below make a lot of compelling arguments; I think you'll need to give some more justification on this point.